.fixtures.yml

```diff
@@ -4,8 +4,8 @@ fixtures:
       repo:      'https://github.com/Ericsson/puppet-module-nisclient.git'
       ref:       'v2.0.0'
     'nsswitch':
-      repo:      'https://github.com/ghoneycutt/puppet-module-nsswitch.git'
-      ref:       'v1.0.0'
+      repo:      'https://github.com/kodguru/puppet-nsswitch.git'
+      ref:       'v3.1.0-forked'
```

Comment:
I did start to drop version numbers here to make sure to always use the latest version while developing.